packages/react-components/src/components/VideoGallery.test.tsx

```diff
@@ -156,6 +156,129 @@ describe('VideoGallery default layout', () => {
   });
 });
 
+describe('VideoGallery floating local video layout', () => {
+  beforeAll(() => {
+    registerIcons({
+      icons: {
+        horizontalgalleryleftbutton: <></>,
+        horizontalgalleryrightbutton: <></>
+      }
+    });
+
+    // Need to mock this because the HorizontalGallery uses this function. But JSDOM does not actually do any
```

Comment:
Now that we're adding these mocks twice, for the same reason, let's add a helper that keeps the reasoning etc. in one place.

Reply:
+1